chadrick-d-dev/silver-nest-api
# Initial data structure

### What’s this PR do?
Creates an express api with knex and postgresql for our Silvernest UI, and gets it to display our seed file data.

### Where should the reviewer start?
Start with the first pull request

### How should this be manually tested?
clone down repo, ensure postgresql and knex are installed on your local machine, run npm install, install postico and postgres.app, create server for Silvernest, run `knex migrate:latest` then run `knex seed:run`, then check out the Silvernest server in postico and look for data

### Any background context you want to provide?
We went on journey to create an express server with PostgreSQL and knex
We were finally able to view our seed file data using Postico
